### ISSUE TITLE
No such templates/angular folder found in running

### ISSUE THREAD

**Issue by ghost** [vivekmore/generator-jhipster-nav-element#35]
##### **Overview of the issue**

The folder specified in the title is not found.

##### **Motivation / Use Case**

The motivation is for [line](https://github.com/vivekmore/generator-jhipster-nav-element/blob/516f6f5827cb58845eec7ea14f86f60fa6e4047f/generators/app/about-us/about-us-ng-template-writer.js#L51) that used the templates folder.
But the npm package published does not contains that folder:
https://github.com/vivekmore/generator-jhipster-nav-element/blob/develop/package.json#L24

##### **Reproduce the error**

```
? What would you like to generate? (More components will be added soon! Stay tuned...) A Simple Page (with a correspondi
ng nav element)
? What would you like the navigation element id to be? (e.g. aboutUs, about_us) hi_there
------------------------------------------------------------
baseName=mytestionicapp
packageName=com.mycompany.myapp
angularAppName=undefined
enableTranslation=false
navElementKey=hi_there
------------------------------------------------------------
Error jhipster-nav-element

Copying template ../../../templates/angular/about-us/src/main/webapp/app/element/element.component.html.ejs failed. [Error: ENOENT: no such file or directory, open 'C:\Users\amanganiello\Desktop\test-ionic\node_modules\generator-jhipster-nav-element\templates\angular\about-us\src\main\webapp\app\element\element.component.html.ejs']

```

##### **Suggest a Fix**

It is simple, in the:
https://github.com/vivekmore/generator-jhipster-nav-element/blob/develop/package.json#L24

Add **templates** includes.

##### **JHipster Version(s)**

JHipster: 6.6.0

##### **Nav Element Version(s)**

v2.3.1

**Comment by ghost** [comment]
I have fixed in #36 moving templates folder under generators/app (it's more correct following yeoman standard)

**Comment by vivekmore** [comment]
Thank you for bringing this to my attention.
Unfortunately your PR has more stuff than the fix.
If you're going to submit a PR please consider keeping it's scope limited to one thing - this helps me review it better and quicker

**Comment by vivekmore** [comment]
This issue is fixed in v2.3.2
